tests/e2e/e2e_test.go

```diff
@@ -381,6 +381,13 @@ var _ = Describe("odoe2e", func() {
 				runCmd("odo push --local " + tmpDir + "/nodejs-ex")
 			})
 
+			It("should be able to push changes while showing logging", func() {
+				// Push the changes with --show-log
+				getLogging := runCmd("odo push --show-log --local " + tmpDir + "/nodejs-ex")
+
+				Expect(getLogging).To(ContainSubstring(" •  Waiting for build to finish  ..."))
```

Comment:
Bullet point at the beginning and three dots at the end looks premature for a exact string validation 

``` •  Waiting for build to finish  ...``` -> ```Waiting for build to finish```

Reply:
CI fails in thsi step because the substring we are looking for ```• Waiting for build to finish ...``` or ```Waiting for build to finish``` is not matching with the actual output.